db/migrate/20201014171538_add_journalist_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddJournalistToArticles < ActiveRecord::Migration[6.0]
+  def change
+    add_column :articles, :journalist_id, :integer
```

Comment:
you need to add this to the articles model

```
  belongs_to :journalist, class_name: 'User'

```

Reply:
Done!